src/util.h

```diff
@@ -29,6 +29,8 @@
 #include <boost/thread/exceptions.hpp>
 
 static const bool DEFAULT_LOGTIMEMICROS = false;
+static const bool DEFAULT_LOGIPS        = false;
+static const bool DEFAULT_LOGTIMESTAMPS = true;
```

Comment:
Never mind. The fLogTimestamps variable was initialized to false before, but the default setting was true. This makes it more consistent. Good.